Srcs/heiftojpeg/heiftojpeg.cpp

```diff
@@ -130,7 +130,7 @@ static void processFile(char *filename, char *outputFileName)
     gridItem = reader.getItemGrid(contextId, gridItemId);
 
     if (VERBOSE) {
-        cout << "grid is " << gridItem.outputWidth << "x" << gridItem.outputHeight << " pixels in tiles " << gridItem.rowsMinusOne << "x" << gridItem.columnsMinusOne << endl;
+        cout << "grid is " << gridItem.outputWidth << "x" << gridItem.outputHeight << " pixels in tiles " << std::to_string(gridItem.columnsMinusOne + 1) << "x" << std::to_string(gridItem.rowsMinusOne + 1) << endl;
```

Comment:
we `use namespace std;` so you don't need the `std::`

Reply:
Ah, sorry for missing that. I see you addressed this in a follow-up commit. 👍